contracts/contracts/token/TrustlinesNetworkToken.sol

```diff
@@ -35,10 +36,21 @@ contract TrustlinesNetworkToken {
         _mint(preMintAddress, preMintAmount);
     }
 
+    /**
+    * @dev Gets the balance of the specified address.
+    * @param owner The address to query the balance of.
+    * @return An uint256 representing the amount owned by the passed address.
+    */
     function balanceOf(address account) public view returns (uint256) {
         return _balances[account];
     }
 
+    /**
+    * @dev Function to check the amount of tokens that an owner allowed to a spender.
```

Comment:
Returns the number of tokens that an owner allowed the spender to transfer with transferFrom?